caphandler/switches.go

```diff
@@ -0,0 +1,103 @@
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+//
+//Licensed under the Apache License, Version 2.0 (the "License"); you may
+//not use this file except in compliance with the License. You may obtain
+//a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//License for the specific language governing permissions and limitations
+// under the License.
+
+//Package caphandler ...
+package caphandler
+
+import (
+	"github.com/ODIM-Project/ODIM/lib-dmtf/model"
+	"github.com/ODIM-Project/PluginCiscoACI/capdata"
+	iris "github.com/kataras/iris/v12"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"strings"
+)
+
+// GetSwitchCollection fetches the switches which are linked to that fabric
+func GetSwitchCollection(ctx iris.Context) {
+	//Get token from Request
+	token := ctx.GetHeader("X-Auth-Token")
```

Comment:
X-Auth-Token is used by plugin or ACI ? FO plugin lets us use basic auth. In a container env we still dont have a mechanism is keep the tokens in sync across plugins

Reply:
X Auth validation condition removed